.github/workflows/nix.yaml

```diff
@@ -12,23 +12,23 @@ jobs:
     - uses: cachix/install-nix-action@v16
     - uses: cachix/cachix-action@v10
       with:
-        name: svthalia-concrexit
-        authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-    - run: nix-build -j 4 --arg version "\"$(git rev-parse HEAD)\"" -A github-actions release.nix
-    - run: nix-shell --run "echo OK"
+        name: svthalia
+        authToken: '${{ secrets.CACHIX_AUTH_TOKEN_SVTHALIA }}'
+    - run: nix build --log-format raw --verbose
     - name: Deploy if this is master or staging
       if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/staging'
+      env:
+        TF_VAR_ssh_private_key: ${{ secrets.STAGING_SSH_PRIVATE_KEY }}
+        TF_VAR_ssh_public_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINgNaleedXX3iDZu8KlafjjURMVwXO+YZU1498tWu2m8 deploykey"
```

Comment:
Not sure what this is, but I think it would be nice to centralize the public key too in secrets?

Reply:
yea that's fair